torch/distributed/elastic/rendezvous/utils.py

```diff
@@ -192,6 +193,8 @@ def __init__(
         *args: Any,
         **kwargs: Any,
     ) -> None:
+        self._name = None
```

Comment:
Can we add name as constructor arg?

Reply:
It is tricky to add it as a constructor parameter due to the open-ended nature of the `__init__` function. I want it to be an optional parameter, but in such case there is no way to distinguish between the callback parameters versus the name parameter. That is in fact why I added a `set_name` function instead of a constructor parameter.